Autoscroll to invited organisation

abhvsn opened the pull request:
## Description
> When user is invited to the organisation existing user should get the invite url with the organisation slug
> When user visits the home page page should be automatically scrolled to the invited organisation    

Fixes #2359 
> if no issue exists, please create an issue and ask the maintainers about this first

## How Has This Been Tested?

> JUnit TCs
> Manual 

1. Sharing organisation to the existing user
2. Sharing organisation to new user

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

sharat87 commented on app/server/appsmith-server/src/test/java/com/appsmith/server/services/UserServiceTest.java:
```diff
@@ -30,11 +30,7 @@
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
Star imports are considered dangerous since they import much more than what we actually want and they do it implicitly. Please configure your IntelliJ so that it doesn't automatically do star imports. Here's a screenshot on how to do it:

![Screenshot 2021-04-09 at 15 01 16](https://user-images.githubusercontent.com/120119/114160726-dd9fdf00-9944-11eb-863e-a4a9d32b20be.png)

sharat87 commented on app/server/appsmith-server/src/main/java/com/appsmith/server/services/UserServiceImpl.java:
```diff
@@ -86,6 +86,10 @@
     // We default the origin header to the production deployment of the client's URL
     private static final String DEFAULT_ORIGIN_HEADER = "https://app.appsmith.com";
 
+    public enum typeOfEmail {welcome, passwordReset, inviteExistingUserToOrg, inviteNewUserToOrg}
+
+    ;
+
```
Looks like this is unused. Can we remove it?

abhvsn replied:
Refactored code pushed.

abhvsn commented:
Refactored code pushed as per above suggestion : 

1. Unused typeOfEmail removed
2. import java.util.* removed with specified imports required (Also modified the intelliJ setting on local machine)

abhvsn commented:
@sharat87 Please review the changes.
